contracts/BaseLaunchpeg.sol

```diff
@@ -98,6 +118,27 @@ abstract contract BaseLaunchpeg is
     /// @param quantity Amount of NFTs minted
     event DevMint(address indexed sender, uint256 quantity);
 
+    /// @dev Emitted on preMint()
+    /// @param sender The address that minted
+    /// @param quantity Amount of NFTs minted
+    /// @param price Price of 1 NFT
+    event PreMint(address indexed sender, uint256 quantity, uint256 price);
+
+    /// @dev Emitted on auctionMint(), batchMint(), allowlistMint(), publicSaleMint()
+    /// @param sender The address that minted
+    /// @param quantity Amount of NFTs minted
+    /// @param price Price in AVAX for the NFTs
+    /// @param startTokenId The token ID of the first minted NFT:
+    /// if `startTokenId` = 100 and `quantity` = 2, `sender` minted 100 and 101
+    /// @param phase The phase in which the mint occurs
+    event Mint(
```

Comment:
re-factored this `Mint()` event by combining the events in `Launchpeg` and `FlatLaunchpeg`